### ISSUE TITLE
Expected indentation of 2 spaces

### ISSUE THREAD

**Issue by ravinggenius** [styled-components/stylelint-processor-styled-components#51]
### Note

I originally opened this as an [issue for stylelint](https://github.com/stylelint/stylelint/issues/2646), but they directed me over here. I have reproduced the issue below. I am using `stylelint-processor-styled-components` v0.1.2.

### Issue

> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I believe this is a bug. When linting [styled-components](https://www.styled-components.com/) styles inside a helper function, indentation is reported incorrectly.

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

```javascript
import { css } from 'styled-components';

const helper = (condition) => {
  if (condition) {
    return css`
      color: red;

      &:hover {
        color: blue;
      }
    `;
  }
  return null;
};
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": "stylelint-config-standard",
  "syntax": "scss"
}
```

> Which version of stylelint are you using?

`7.11.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`yarn run lint` which calls `stylelint 'src/**/*.js?(x)'`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes. Both styled-components and SCSS look for `&` as a placeholder when expanding nested rules.

> What did you expect to happen?

No errors to be reported. An error only gets reported when both the `color: red;` style and the `&:hover` block is present. Removing either does not trigger an `indentation` error to be reported.

> What actually happened (e.g. what warnings or errors you are getting)?

The following errors were flagged:

```shell
Button.jsx
 36:5  ✖  Expected indentation of 2 spaces   indentation
 38:5  ✖  Expected indentation of 2 spaces   indentation
 39:7  ✖  Expected indentation of 4 spaces   indentation
 40:5  ✖  Expected indentation of 2 spaces   indentation
 41:3  ✖  Expected indentation of 0 spaces   indentation
```

**Comment by ravinggenius** [comment]
If I remove the `css` from the template literal, the issue is not reported since the block is no longer recognized as a stylesheet fragment. Removing `css` also impedes my editor's ability to syntax highlight the styles.

**Comment by ismay** [comment]
Thanks for reporting this. To speed up work on this it would be helpful if you could submit a PR with a failing testcase demonstrating this problem. If not we'll take care of it as well but it might take a bit longer. Thanks again for the report!

**Comment by ravinggenius** [comment]
I did the branch and everything locally, but I don't have the correct SSH keys on this computer, so I had to redo everything through GitHub's web interface, which is why I forgot the `css, ` bit and had to make a second commit to the branch.

**Comment by ismay** [comment]
Awesome, thanks for the pr!
